src/bindings/python/src/openvino/runtime/opset9/ops.py

```diff
@@ -146,3 +146,49 @@ def irdft(
         inputs = as_nodes(data, axes, signal_size)
 
     return _get_node_factory_opset9().create("IRDFT", inputs)
+
+
+@nameable_op
+def generate_proposals(
+    im_info: NodeInput,
+    anchors: NodeInput,
+    deltas: NodeInput,
+    scores: NodeInput,
+    min_size: float,
+    nms_threshold: float,
+    post_nms_count: int,
+    pre_nms_count: int,
```

Comment:
Please follow docs when setting attributes order: https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/detection/GenerateProposals_9.md

@meiyang-intel @luo-cheng2021 I tracked C++ implementation back to you. Could you update it to follow docs order of attributes so PyAPI can fallback to it without any issues?

```suggestion
    pre_nms_count: int,
    post_nms_count: int,
```

Reply:
updated